test/wasi/proc_raise.wast

```diff
@@ -0,0 +1,10 @@
+(module
+  (import "wasi_snapshot_preview1" "proc_raise" (func $wasi_proc_raise (param i32) (result i32)))
+
+  (func (export "proc_raise") (result i32)
+    i32.const 0 ;; illegal signal ID
+    call $wasi_proc_raise
```

Comment:
What is the result of `proc_raise` when signal 0 is given?

Reply:
It results 52, which is the value of error _UVWASI_ENOSYS_.

**UVWASI** returns this, if it fails to convert signal code to uv signal.

In the `uvwasi_proc_raise()` function:
```C
  r = uvwasi__translate_to_uv_signal(sig);
  if (r == -1)
    return UVWASI_ENOSYS;
```

Is it good or may I change the code to raise an another signal?

Reply:
What about giving a valid signal and checking the result of UVWASI?
Is there any possible example?

Reply:
I tried several valid/_used_ signals. In those cases, the program terminates with the given signal (e.g.: **SIGQUIT** --> `exit (core dumped)`), but thus the test fails (Walrus returns with non-zero number).